src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -49,9 +49,12 @@ public function process(ContainerBuilder $container)
             $coreThemePath = dirname(dirname($reflClass->getFileName())).'/Resources/views/Form';
             $container->getDefinition('twig.loader.native_filesystem')->addMethodCall('addPath', array($coreThemePath));
 
-            $paths = $container->getDefinition('twig.cache_warmer')->getArgument(2);
-            $paths[$coreThemePath] = null;
-            $container->getDefinition('twig.cache_warmer')->replaceArgument(2, $paths);
+            if ($container->hasDefinition('twig.cache_warmer')) {
+                $paths = $container->getDefinition('twig.cache_warmer')->getArgument(2);
+                $paths[$coreThemePath] = null;
+                $container->getDefinition('twig.cache_warmer')->replaceArgument(2, $paths);
+            }
+
             $container->getDefinition('twig.template_iterator')->replaceArgument(2, $paths);
```

Comment:
variable `$paths` is undefined if `twig.cache_warmer` doesn't exist, it must be included in the new `if` condition.